validator/utils.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+import yaml
+
+try:
+    from yaml import CSafeLoader as SafeLoader
+except ImportError:
+    from yaml import SafeLoader as SafeLoader
+
+
+def parse_anymarkup(filename):
+    file_ext = os.path.splitext(filename)[1][len(os.path.extsep):]
+
+    res = {}
```

Comment:
nit: This is a bit deceiving, since we actually don't know what kind of structure we're going to be returning once we have parsed the yaml or json doc...

Reply:
Returning an empty dict by default seems acceptable in the context. json/yaml loading should either result in an object or fail with an exception.

What would be a preffered way?

Reply:
I don't see this default value being ever returned... But it's fine to have a default value that will evaluate to `False` nonetheless in case this code evolves, so leave it as it is :)